src/VisualStudio/Core/Def/Implementation/ProjectSystem/AbstractProject.cs

```diff
@@ -1220,8 +1220,6 @@ public virtual void Disconnect()
 
                     if (_projectsReferencingMe.Count > 0)
                     {
-                        FatalError.ReportWithoutCrash(new Exception("We still have projects referencing us. That's not expected."));
```

Comment:
This might be worth leaving a comment; otherwise _projectsReferencingMe is all being updated for no reason.

Reply:
I've added a comment.